frontend/src/styles/Error.css

```diff
@@ -0,0 +1,15 @@
+.error_image {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 200px;
+    width: 250px;
+}
+
+.error_message {
+    text-align: center;
+    margin-top: 10px;
+    font-size: 35px;
+    color:  #1DE28F;
+    font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen-Sans, Ubuntu, Cantarell, Helvetica Neue, sans-serif;
```

Comment:
What's this mess?

Reply:
using the same font cascade as Jay for the profile page, specifying backups?